main.go

```diff
@@ -39,11 +39,13 @@ var (
 	disableEventarcCollector         = flag.Bool("collector.eventarc.disable", false, "Disables the metrics collector for Cloud Eventarc")
 	disableFunctionsCollector        = flag.Bool("collector.functions.disable", false, "Disables the metrics collector for Cloud Functions")
 	disableIAMCollector              = flag.Bool("collector.iam.disable", false, "Disables the metrics collector for Cloud IAM")
-	disableKubernetesCollector       = flag.Bool("collector.kubernetes.disable", false, "Disables the metrics collector for GKE")
+	disableKubernetesCollector       = flag.Bool("collector.kubernetes.disable", false, "Disables the metrics collector for Google Kubernetes Engine (GKE)")
 	disableLoggingCollector          = flag.Bool("collector.logging.disable", false, "Disables the metrics collector for Cloud Logging")
 	disableMonitoringCollector       = flag.Bool("collector.monitoring.disable", false, "Disables the metrics collector for Cloud Monitoring")
 	disableSchedulerCollector        = flag.Bool("collector.scheduler.disable", false, "Disables the metrics collector for Cloud Scheduler")
 	disableStorageCollector          = flag.Bool("collector.storage.disable", false, "Disables the metrics collector for Cloud Storage")
+
+	ConfigKubernetesCollector				 = flag.String("collector.kubernetes.config", `{"enableClusterAndNodePoolInfoMetric": false}`, "Specifies specific settings for the collector")
```

Comment:
1. r/`ConfigKubernetesCollector`/`configKubernetesCollector`
2. `collector.gke.disable`

Reply:
There's a potential for conflict: `--disableKubernetesCollector --configKubernetesCollector="{}"`

Per previous comment, it's unclear why `--configKubernetesCollector` isn't another bool but the values of these flags should be checked after parsing and a message logged if they conflict.

Reply:
The goal is that other configurations can be defined later. So I'm paving the way for that.

Reply:
My intention is to do something similar to kube-state-metrics, where we can define specific configurations per collector.

Ref: https://github.com/kubernetes/kube-state-metrics/blob/main/pkg/options/options.go#L164

It might make more sense to use Yaml's approach.

I'll make a proposal for change and you can tell me what you think.

Reply:
I'll simplify it and use a Boolean.

We'll think about how to pass other configurations later.